app.py

```diff
@@ -5,6 +5,14 @@
 
 from aws_change_observer.aws_change_observer_stack import AwsChangeObserverStack
 
+from dotenv import load_dotenv
+
+load_dotenv()
+
+region = os.getenv("REGION")
+account = os.getenv("ACCOUNT")
+is_prod = os.getenv("PROD") == 'True'  # Convert string to boolean if needed
```

Comment:
@PCBush 
create a .env file in the project's root directory and set above variables there